.gitignore

```diff
@@ -5,3 +5,5 @@
 .idea
 *.iml
 plugin_tools
+
+*.orig
```

Comment:
It's from git mergetool. It defaults to creating backup files after resolving a git conflict.